src/pages/registration/resource_type_tab/sub_type_forms/artistic_types/music_performance/ConcertModal.tsx

```diff
@@ -45,7 +45,7 @@ const emptyConcert: Concert = {
   extent: '',
   description: '',
   concertProgramme: [],
-  partOfSeries: false,
+  concertSeries: '',
```

Comment:
Tom streng betyr vel at boksen vil være checked nå? Så nå vil den være checked by default? Jeg ville antatt at det motsatte er mer naturlig, og at vi heller kan sette `undefined` her
```suggestion
  concertSeries: undefined,
```